src/customers/repository/customers.repository.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import Customer from '../domain/customer.domain';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export default class CustomersRepository {
```

Comment:
For now, you can remove the unused methods (`findAll` and `remove`). To keep the code clean, we should create only methods we need. 

You can also add the `async` keyword to all methods. It's a best practice to mark functions that return a Promise as async.

This comment applies to other files too.

Reply:
done